website/src/pages/showcase.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import clsx from 'clsx';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import styles from '../pages/showcase.module.css';
+import StandWithUkraine from '../components/StandWithUkraine';
```

Comment:
You could use `@site/src/components/StandWithUkraine` instead.

Reply:
Learned more about the usage of this @site and have changed all of the references and to images using it now.